index.html

```diff
@@ -1479,9 +1478,9 @@
                 <li>Set the <a>presentation session identifier</a> of
                 <var>S</var> to <var>I</var>.
                 </li>
-                <li>Establish the connection between the controlling and
-                presenting browsing contexts using an implementation specific
-                mechanism.
+                <li>Establish the connection between the controlling and <a data-lt=
```

Comment:
Maybe define this term above as "receiving browsing context|receiving browsing contexts", similar to the controlling case.